src/handlers/ban.ts

```diff
@@ -12,82 +12,100 @@ module.exports = {
 
         const fetchedLogs = await msg.guild.fetchAuditLogs({
             limit: 1,
-            type: 'MEMBER_BAN_ADD',
+            type: AuditLogEvent.MemberBanAdd,
         });
 
         const modLogsChannel = msg.guild.channels.resolve(Channels.MOD_LOGS) as TextChannel | null;
 
         const banLog = fetchedLogs.entries.first();
 
-        if (modLogsChannel && !ModLogsExclude.some((e) => e)) {
-            const noLogEmbed = makeEmbed({
-                color: Colors.Red,
-                author: {
-                    name: `[BANNED] ${msg.user.tag}`,
-                    iconURL: msg.user.displayAvatarURL(),
-                },
-                description: `${msg.user.tag} was banned from ${msg.guild.name} but no audit log could be found.`,
-                footer: { text: `User ID: ${msg.user.id}` },
-            });
-
-            if (!banLog) await modLogsChannel.send({ embeds: [noLogEmbed] });
-        }
-
         const {
             reason,
             executor,
             target,
         } = banLog;
 
+        const noLogEmbed = makeEmbed({
+            color: Colors.Red,
+            author: {
+                name: `[BANNED] ${msg.user.tag}`,
+                iconURL: msg.user.displayAvatarURL(),
+            },
+            description: makeLines([
+                `${msg.user.tag} was banned from ${msg.guild.name} but no audit log could be found.`,
+                '',
+                '**NOTE - This was a non bot ban.**',
```

Comment:
Prefer
```suggestion
                bold('This was a non bot ban.'),
```
or the way it is?